peekaboo/toolbox/ms_office.py

```diff
@@ -59,3 +59,34 @@ def has_office_macros(office_file):
     except Exception as error:
         logger.exception(error)
         return False
+
+def has_office_macros_with_auto_action(office_file):
+    """
+    Detects macros for Auto_ actions in Microsoft Office documents.
+
+    @param office_file: The MS Office document to check for Auto_ macros.
+    @return: True if Auto_ macros where found, otherwise False.
+             If VBA_Parser crashes it returns False too.
+    """
+
+    SUSPICIOUS_KEYWORDS = ["AutoOpen", "AutoClose"]
+
+    file_extension = office_file.split('.')[-1]
+    if file_extension not in MS_OFFICE_EXTENSIONS:
+        print("/%s/" % file_extension)
+        return False
+    try:
+        # VBA_Parser reports macros for office documents
+        vbaparser = VBA_Parser(office_file)
+        for (filename, stream_path, vba_filename, vba_code) in vbaparser.extract_macros():
+            for w in  SUSPICIOUS_KEYWORDS:
+                if w in vba_code:
+                    return True
+
+        return vbaparser.detect_vba_macros()
```

Comment:
Shouldn't this be `return False`? Otherwise this still returns True if there's *any* macro in the file, causing a false positive.

Reply:
Fixed with latest commits.